wagtail/contrib/search_promotions/tests.py

```diff
@@ -154,6 +154,70 @@ def test_results_are_ordered_alphabetically(self):
         # "aargh snake" should be the first result alphabetically
         self.assertEqual(response.context['queries'][0].query_string, "aaargh snake")
 
+    def test_results_ordering(self):
+        self.make_search_picks()
+        url = reverse('wagtailsearchpromotions:index')
+        SearchPromotion.objects.create(
+            query=Query.get("zyzzyvas"),
+            page_id=1,
+            sort_order=0,
+            description="no definition found, this is a valid scrabble word though",
+        )
+
+        SearchPromotion.objects.create(
+            query=Query.get("aardwolf"),
+            page_id=1,
+            sort_order=22,
+            description="Striped hyena of southeast Africa that feeds chiefly on insects",
+        )
+
+        popularQuery = Query.get("optimal")
+        for i in range(50):
+            popularQuery.add_hit()
+        SearchPromotion.objects.create(
+            query=popularQuery,
+            page_id=1,
+            sort_order=15,
+            description="An odly popular search term?",
+        )
+
+        popularQuery = Query.get("sub-optimal")
```

Comment:
Looks like this needs to be changed to match what the tests expect (or vice-versa):

```suggestion
        popularQuery = Query.get("suboptimal")
```

Reply:
Sounds possibly related to #5416 / #5418 - perhaps this test was written before that change was committed?